chain/client/src/sync/state.rs

```diff
@@ -749,20 +729,23 @@ impl StateSync {
             //    in the previous (or about a subset of them), return success right away
```

Comment:
`tracking_shards` is misleading here because it actually holds the shards that the node needs to sync not all the tracked shards.